dataactbroker/handlers/fileHandler.py

```diff
@@ -488,18 +491,25 @@ def call_d_file_api(self, file_type_name, cgac_code, start_date, end_date, job,
         else:
             self.complete_generation(task_key, file_type)
 
-    def download_file(self, local_file_path, file_url):
+    def download_file(self, local_file_path, file_url, upload_name, response):
         """ Download a file locally from the specified URL, returns True if successful """
         if not self.isLocal:
-            with open(local_file_path, "w") as file:
-                # get request
-                response = requests.get(file_url)
-                if response.status_code != 200:
-                    # Could not download the file, return False
-                    return False
-                # write to file
+            bucket = CONFIG_BROKER['aws_bucket']
+            region = CONFIG_BROKER['aws_region']
+            conn = boto.s3.connect_to_region(region).get_bucket(bucket).new_key(upload_name)
+            with smart_open.smart_open(conn, 'w') as writer:
+                # get request if it doesn't already exist
+                if not response:
+                    response = requests.get(file_url, stream=True)
+                    # we only need to run this check if we haven't already
+                    if response.status_code != 200:
+                        # Could not download the file, return False
+                        return False
+                # write (stream) to file
                 response.encoding = "utf-8"
-                file.write(response.text)
+                for chunk in response.iter_content(chunk_size=1024):
```

Comment:
Can you put something in the comments why the chunk size is set to 1024? Also, we may want to make it a constant variable (so just an all caps variable below the imports) so it's easily manipulated.